f2/src/locales/fr.json

```diff
@@ -122,7 +122,7 @@
   "contactInfoPage.victimDetail": "Détails sur la victime",
   "contactinfoForm.email.warning": "Entrez une adresse courriel valide.",
   "contactinfoForm.fullName.warning": "Entrez votre nom complet",
-  "contactinfoForm.phone.warning": "Entrez un numéro de téléphone de 10 chiffres.",
+  "contactinfoForm.phone.warning": "Entrez un numéro de téléphone valide.",
```

Comment:
Where did this French translation come from?

Reply:
I translated it. Where should usually the French translation come from?